dist/platforms/mac/steps/activate.sh

```diff
@@ -2,6 +2,9 @@
 
 echo "Requesting activation"
 
+# Need this because it tries to initialize the library when activating
+UNITY_PROJECT_PATH="$GITHUB_WORKSPACE/$PROJECT_PATH"
```

Comment:
Not sure if you got that part, but we already work around this in the Ubuntu activate file.

Passing a projectPath did not work as expected because:
- if you pass an empty path, it won't recognise it as a project
- if you pass the real project, it'll initialise library during this step
- if you pass no parameter and you're in the default folder while the project is checked out in that folder too, it'll also initialise the library during this step

The only way we managed to make this work consistently was to
- move to a different path
- use no parameter so it uses the default behaviour (which is creating a new project which doesn't need initialising) in that different path
- switch back to the previous path.